addon/-private/execution_context/rfc268.js

```diff
@@ -0,0 +1,166 @@
+import Ember from 'ember';
+import {
+  guardMultiple,
+  buildSelector,
+  findClosestValue,
+  getRoot
+} from '../helpers';
+import {
+  getContext,
+  visit,
+  click,
+  fillIn,
+  triggerEvent
+} from '@ember/test-helpers';
+import {
+  ELEMENT_NOT_FOUND,
+  throwBetterError
+} from '../better-errors';
+import Ceibo from 'ceibo';
+
+const { $, RSVP: { resolve } } = Ember;
```

Comment:
We should not use `Ember.$` in the `e-c-p-o` internally. That would not allow to use this contexts in the app with disabled `jquery`.

It can be fixed by importing an isolated jquery instance:
https://github.com/san650/ember-cli-page-object/blob/57183487aa9a8446861f4fbc52c03dbc02eeb1e6/addon/-private/execution_context/native-events-context.js#L1

Reply:
Also we can use new modules imports.
```
import { resolve } from 'rsvp';
```

Reply:
Yeah, I was wondering about that -- should I just run the codemod to migrate everything to the new module imports?

Reply:
Sure, i think it quickest way to do it